erpcgen/src/templates/c_coders.template

```diff
@@ -232,7 +232,7 @@ codec->readData({$info.name}, {$info.sizeTemp} * sizeof({$info.builtinTypeName})
 {# Encode sending data #}
 {% def encodeBuiltinType(info) -----------------  %}
 {% if info.builtinType == "kStringType" %}
-codec->writeString(strlen({$info.name}), {$info.name});
+codec->writeString(strlen((const char*){$info.name}), (const char*){$info.name});
```

Comment:
if info.builtinType == "kStringType" looks like it is not counting with ustring type. Am i wrong?

Reply:
Ok i looked into code and it should be ok.